### ISSUE TITLE
Define custom sections into .clog.toml file

### ISSUE THREAD

**Issue by fedexin40** [clog-tool/clog-cli#101]
Hi guys I am stuck trying to setup the .clog.toml file to add some customs sections, I dont know what I am doing wrong.

The file looks like 

`[clog]
repository = "http://gitlab.openpyme.mx/tools/ea_import.git"
outfile = "CHANGELOG.md"
from-latest-tag = true

[sections]
Performance = ["perf", "impr", "im", "imp"]
"Bug Fixes" = ["fix"]
"Code refactor" = ["refactor"]
Feature = ["feat"]
`

First I placed the file in ~/.clog.toml and I used the instruction to create the file
`clog -c ~/.clog.toml -C ea_import/CHANGES.md -f 3dcdddf612d8b8ab91b1e6546e9e2c824b5c04d0 --setversion 1.1.0
`
After that I put the file into the same directory than my repository but the result is always the same 
`<a name="1.1.0"></a>
## 1.1.0 (2017-04-03)`

I dont know what I am doing wrong, using default setting all work well but whitout having "Code refactor" commits.

Could you help or guide me please?

**Comment by kbknapp** [comment]
Try removing the `from-latest-tag` line. Does it still only show a header?

**Comment by fedexin40** [comment]
Yes I have the same result, all work okay if I dont use a custom configuration.

**Comment by fedexin40** [comment]
After a while looking aroung 
`clog -c ~/.clog.toml -g . -C ea_import/CHANGES.md -f 3dcdddf612d8b8ab91b1e6546e9e2c824b5c04d0 --setversion 1.1.0`

I need to use `--git-dir` in somewhere
